style.css

```diff
@@ -331,10 +331,12 @@ h1 span {
   margin: 30px;
   background: #36b37f;
   border-radius: 4px;
+  outline: none;
+  border: none;
 }
 
 .btn-form:hover {
-  opacity: 0.9;
+  color: #36b37f;
```

Comment:
- [ ] Great job implementing the buttons, but right now the `:hover` action is not working because is has the same style as the regular button, if you check the original design, for the action it has a `box-shadow`, I suggest that you implement it to match the Figma design perfectly, you can check it here:

![image](https://user-images.githubusercontent.com/66967274/144235275-3bc3ff05-2e73-4bbb-a884-f56b3a6ce919.png)

Reply:
Hi Vanessa, 
I have implemented all the changes you asked me to change here and it's working. I therefore humbly ask that you give it a thorough review.
Best regards,
Thanks.